Gruntfile.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = function (grunt) {
+
+    // Load external grunt task config.
+    grunt.loadTasks('./grunt');
+
+    grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-jscs');
```

Comment:
Also since its my fist time doing this sort of thing. id prefer to get it all working first then improve it.
if i did it from the start id possibly be confused and not understand it.